test/common/pixel-tests

```diff
@@ -81,11 +81,59 @@ cmd_reset() {
     cmd_pull
 }
 
+pixel_test_logs_url() {
+    arg=${1:-}
+
+    if [[ "$arg" == http* ]]; then
+        echo $arg
+        return
+    fi
+
+    repo=$(git remote get-url origin | sed -e 's,\(git@github.com:\|https://github.com/)\),,'  -e 's,\.git$,,')
```

Comment:
This doesn't seem to work as intended:
```
❱❱❱ git remote get-url origin
https://github.com/cockpit-project/cockpit.git
❱❱❱ git remote get-url originw
git@github.com:cockpit-project/cockpit.git

❱❱❱ git remote get-url origin | sed -e 's,\(git@github.com:\|https://github.com/)\),,' -e 's,\.git$,,'
https://github.com/cockpit-project/cockpit

❱❱❱ git remote get-url originw | sed -e 's,\(git@github.com:\|https://github.com/)\),,' -e 's,\.git$,,'
cockpit-project/cockpit
```

I.e. the https case is broken due to the extra `)`. I recommend to use `-r` to avoid the backslashery, and simplify this:

    sed -re 's,git@github.com:|https://github.com/,,' -e 's,\.git$,,'

This works for me in both cases.

Reply:
Ouch, thanks!